### PR TITLE
Update 02-preparing-your-project-being-open-sourced.md

### PR DESCRIPTION
**Pull request by Idadelveloper**

### CODE REVIEW

**Comment by frinyvonnick** [comment]
Hi @Idadelveloper, Thank you for your contribution 🙏 

We store french translations in another folder called [fr](https://github.com/zenika-open-source/open-source-white-paper/tree/master/fr). Could you create a file for your translation in this folder and let the english file unchanged ?

I'll do a review as soon as possible 👍

File: en/02-preparing-your-project-being-open-sourced.md [hunk 1]
```diff
@@ -1,70 +1,67 @@
...
 
-The most famous code of conduct is [Contributor Covenant](https://www.contributor-covenant.org/). It is used by thousands of open source projects. If your project is on GitHub you can directly add Contributor Covenant or [Citizen Code of Conduct](http://citizencodeofconduct.org/) through your repository interface. You need to go to `Insights > Community > Code of conduct`, then by clicking on the Add button and choosing a code of conduct, it will create a commit for you.
+Vous devez également préciser la manière de signaler une violation, par exemple par courrier électronique.
+Le code de conduite le plus connu est [contributeur](https://www.contributor-covenant.org/). Il est utilisé par des milliers de projets open source. Si votre projet est sur GitHub, vous pouvez directement ajouter le covenant du contributeur ou le [code de conduite du citoyen].(http://citizencodeofconduct.org/) via votre interface de référentiel. Vous devez aller dans «Insights> Communauté> Code de conduite», puis en cliquant sur le bouton Ajouter et en choisissant un code de conduite, un commit sera créé pour vous.
```
Comment (geromegrignon) [inline en/02-preparing-your-project-being-open-sourced.md]: Contributor Covenant et Citizen Code of Conduct being organizations name, keep them in english.

File: en/02-preparing-your-project-being-open-sourced.md [hunk 2]
```diff
@@ -1,70 +1,67 @@
...
+Si votre projet est sur GitHub, ce fichier sera [lié automatiquement](https://help.github.com/en/articles/setting-guidelines-for-repository-contributors) lorsqu'un contributeur ouvre un problème ou crée une demande de tirage.
 
 ## Code of conduct
 
-Having a CODE_OF_CONDUCT.md in your public repository lets potential _contributors_ know in advance how they can expect to be treated by the community and _maintainers_. -- [Michael Jolley](https://dev.to/michaeljolley/using-a-contributing-codeofconduct-to-assist-others-in-contributing-to-public-repositories-1l90)
-
-The code of conduct of your project is a document that protects every participant. It helps creating a welcoming community.
+Le fait d'avoir un CODE_OF_CONDUCT.md dans votre référentiel public permet aux potentiels contributeurs de savoir à l'avance comment ils peuvent s'attendre à être traités par la communauté et les mainteneurs. - [Michael Jolley](https://dev.to/michaeljolley/using-a-contributing-codeofconduct-to-assist-others-in-contributing-to-public-repositories-1l90)
```
Comment (geromegrignon) [inline en/02-preparing-your-project-being-open-sourced.md]: replace 'référentiel' by 'répertoire'

File: en/02-preparing-your-project-being-open-sourced.md [hunk 3]
```diff
@@ -1,70 +1,67 @@
...
 
-The contributing guide is also the right place to describe coding styles. You can enforce good practices such as testing or conventions like in the [Immutadot's contributing guide](https://github.com/zenika-open-source/immutadot/blob/master/.github/CONTRIBUTING.md#tests-and-code-style-policeman).
+Si vous voulez des contributeurs de code, vous devez fournir tous les détails sur la configuration de l'environnement de développement du projet et sur la manière de soumettre une contribution. [Guide de contribution de Mocha](https://github.com/mochajs/mocha/blob/master/.github/CONTRIBUTING.md#shoe-contributing-code-step-by-step) est un bon exemple d'instructions étape par étape pour obtenir votre contribution fusionnée.
+Le guide de contribution est également le bon endroit pour décrire les styles de codage. Vous pouvez appliquer les bonnes pratiques telles que les tests ou les conventions comme dans le [guide de contribution d'Immutadot](https://github.com/zenika-open-source/immutadot/blob/master/.github/CONTRIBUTING.md#tests-and-code-style-policeman).
```
Comment (geromegrignon) [inline en/02-preparing-your-project-being-open-sourced.md]: replace 'appliquer les bonnes pratiques' by 'imposer des bonnes pratiques'

File: en/02-preparing-your-project-being-open-sourced.md [hunk 4]
```diff
@@ -1,70 +1,67 @@
...
+Un fichier CONTRIBUTING.md, situé dans votre référentiel ou site open source, fournit aux contributeurs potentiels à un projet un guide expliquant comment ils peuvent vous aider dans votre projet ou votre groupe d'étude. Il est de règle de mettre le mot "contribuer" en majuscule en tant que titre du fichier et de l'enregistrer en tant que ressource dans markdown (d'où l'extension .md). - [laboratoire scientifique Mozilla](https://mozillascience.github.io/working-open-workshop/contributing/)
 
-Most of the time contributions are made by users because they encounter an issue using your project. It's nice to have instructions on how to report a bug or suggest a new feature.
+Plus tôt, nous avions parlé de "l'entonnoir des contributeurs open source" dans notre fichier Lisez-moi, nous avons fourni de la documentation pour aider les utilisateurs à installer et à utiliser votre projet. Nous allons maintenant nous concentrer sur les "contributeurs". Le guide de contribution est conçu pour donner des instructions à tous ceux qui souhaitent participer à votre projet.
```
Comment (geromegrignon) [inline en/02-preparing-your-project-being-open-sourced.md]: don't translate 'Readme'.
Make sure to keep markdown styling with italic on 'utilisateurs' and 'contributeurs'

File: en/02-preparing-your-project-being-open-sourced.md [hunk 5]
```diff
@@ -1,70 +1,67 @@
...
-
-Earlier we talked about "The open source contributor funnel" in our readme, we provided documentation to help _users_ installing and using your project, we will now focus on _contributors_. The contributing guide is designed to give instructions to everybody that wants to participate to your project.
+Un fichier CONTRIBUTING.md, situé dans votre référentiel ou site open source, fournit aux contributeurs potentiels à un projet un guide expliquant comment ils peuvent vous aider dans votre projet ou votre groupe d'étude. Il est de règle de mettre le mot "contribuer" en majuscule en tant que titre du fichier et de l'enregistrer en tant que ressource dans markdown (d'où l'extension .md). - [laboratoire scientifique Mozilla](https://mozillascience.github.io/working-open-workshop/contributing/)
```
Comment (geromegrignon) [inline en/02-preparing-your-project-being-open-sourced.md]: replace ' situé dans votre référentiel ou site open source' by 'situé dans votre répertoire open source ou sur votre site'

replace 'contributeurs potentiels à un projet' by 'contributeurs potentiels d'un projet'

File: en/02-preparing-your-project-being-open-sourced.md [hunk 6]
```diff
@@ -1,70 +1,67 @@
...
 
-Your project description and first paragraph of your readme should be a simple summary of your project which hits all the important keywords that people search for. -- [Andrey Petrov](https://medium.com/code-zen/how-to-maintain-a-successful-open-source-project-aaa2a5437d3a)
+La description de votre projet et le premier paragraphe de votre fichier Lisez-moi doivent constituer un résumé simple de votre projet, qui concerne tous les mots-clés importants recherchés par les internautes. - [Andrey Petrov](https://medium.com/code-zen/how-to-maintain-a-successful-open-source-project-aaa2a5437d3a)
 
-First of all, nobody is in your head. You should explain the goal of your project. It doesn't have to be an essay, a single sentence is good enough like in [Conference Hall’s readme](https://github.com/bpetetot/conference-hall).
+Tout d’abord, personne n’est dans votre tête. Vous devriez expliquer le but de votre projet. Il n’est pas nécessaire que ce soit un essai, une seule phrase suffit, comme dans le [readme de la salle de conférence](https://github.com/bpetetot/conference-hall).
```
Comment (geromegrignon) [inline en/02-preparing-your-project-being-open-sourced.md]: Keep Conference Hall in english as it's the project official name.

**Comment by frinyvonnick** [comment]
I removed the changes on the english version, thank you for your work @Idadelveloper 👏

**Pull request merged**